virttest/migration.py

```diff
@@ -270,27 +282,159 @@ def do_migration(self, vms, srcuri, desturi, migration_type,
         :param options: migration options
         :param thread_timeout: time out seconds for the migration thread running
         :param ignore_status: determine if an exception is raised for errors
-        :param func: the function executed during migration thread is running
+        :param func: the function or list of functions executed during migration
```

Comment:
How about adding one more argument for these funcs? It's not easy to understand the meaning if it has multiple types I think.

Reply:
updated by adding a new parameter.